spec/spec_helper_acceptance_local.rb

```diff
@@ -93,7 +93,7 @@ def set_hieradata(hieradata)
 def clear_temp_hieradata
   if @temp_hieradata_dirs && !@temp_hieradata_dirs.empty?
     @temp_hieradata_dirs.each do |data_dir|
-      if File.exists?(data_dir)
```

Comment:
preparation for Ruby 3.2.2 :p https://www.reddit.com/r/ruby/comments/1196wti/psa_and_a_little_rant_fileexists_direxists/
https://stackoverflow.com/questions/48192762/did-ruby-deprecate-the-wrong-file-exists-method